import_export/resources.py

```diff
@@ -269,6 +276,31 @@ def get_or_init_instance(self, instance_loader, row):
         else:
             return (self.init_instance(row), True)
 
+    def validate_instance(self, instance, import_validation_errors={}, validate_unique=True):
```

Comment:
```
def validate_instance(self, instance, import_validation_errors=None, validate_unique=True):
    if import_validation_errors is None:
        errors = {}
    else:
        erros = import_validation_errors.copy()
```

Suggestion - using None instead of mutable `dict`. 
As per https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

Reply:
@bmihelac Fully agree - that's a better solution :)